.github/workflows/book.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - run: mdbook build -d public
         working-directory: book
-      - run: echo "DEST=$(if [ "$GITHUB_EVENT_NAME" == "release" ] || [ "$GITHUB_EVENT_NAME" == "workflow_dispatch" ]; then echo 'stable/latest'; else echo 'git'; fi)" >> ${GITHUB_ENV}
+      - run: echo "DEST=$(if [ "$GITHUB_EVENT_NAME" == "workflow_dispatch" ]; then echo 'git'; else echo 'stable/latest'; fi)" >> ${GITHUB_ENV}
```

Comment:
I think you meant the opposite here, now every commit will be published to latest/stable and only the workflow_dispatch (manually triggered) gets published to git which is weird....

Reply:
I do want that every commit gets pushed to stable from now on. I do depend on stable after all.

It is true though, that we can get rid of workflow_dispatch for now